package.json

```diff
@@ -75,6 +76,7 @@
     "scroll-to": "0.0.2",
     "semver": "^5.1.0",
     "weex-vue-framework": "2.0.5-weex.1",
+    "weex-rx-framework": "0.1.13",
```

Comment:
I dont think so, `build:frameworks:config` will scan `package.json` file, the dep of weex-rx-framework is part of the commit.